charts/rancher-vsphere-cpi/values.yaml

```diff
@@ -13,6 +13,7 @@ vCenter:
   labels:
     region: "k8s-region"
     zone: "k8s-zone"
+    generate: true
```

Comment:
Same as the other comment regarding the default - I think we want this to be `false` by default here.